src/components/Api.js

```diff
@@ -0,0 +1,220 @@
+export default class Api {
+  constructor(options) {
+    this._url = options.url;
+    this.authorization = options.authorization;
+  }
+
+  getCurrentUser(endPoint, DOMLocations) {
+    return fetch(`${this._url}${endPoint}`, {
+      method: "GET",
+      headers: {
```

Comment:
Needs correction: You should store the whole headers object in the constructor and reuse it in all the methods

```js
method: "GET",
headers: this._headers
```

Reply:
Done